run_all_experiments.py

```diff
@@ -50,11 +50,13 @@
 BENCHMARK_DIR: str = f'{BENCHMARK_ROOT}/comparison'
 RESULTS_DIR: str = run_one_experiment.RESULTS_DIR
 GENERATED_BENCHMARK: str = 'generated-benchmark-'
+PROJECT_SUMMARY_JSON = os.path.join(RESULTS_DIR, 'project_coverage_gain.json')
```

Comment:
Mirroring the comment thread from here: https://github.com/google/oss-fuzz-gen/pull/491#discussion_r1681887825 so it does not get lost

CC @erfanio @DonggeLiu

Reply:
Thanks, I was a bit unsure about the scope of this PR: Is there a follow-up PR (e.g., to organize structure or [add other info](https://github.com/google/oss-fuzz-gen/pull/491#discussion_r1683362565) to this file)?
I don't mind skipping 105 for now, but a more extensible structure is certainly preferred in this PR.